src/content/docs/browser/browser-monitoring/troubleshooting/troubleshoot-your-browser-monitoring-installation.mdx

```diff
@@ -229,7 +211,7 @@ These troubleshooting steps apply to problems when New Relic's browser monitorin
     </html>
     ```
 
-    If the Javascript is correctly placed, compare it with the original snippet you generated to ensure it is complete and identical. Incomplete or modified snippets may not report correctly.
+    If the Javascript is correctly placed, compare it in the final HTML served to your users with the original pasted snippet to ensure it's complete and identical. Incomplete or modified snippets may not report correctly. This could happen with frameworks that try to sanitize or alter raw javascript rendered without using the appropriate method.
```

Comment:
```suggestion
    If the Javascript is correctly placed, compare the final HTML served to your users with the original pasted snippet to ensure it's complete and identical. Incomplete or modified snippets may not report correctly. This could happen with frameworks that try to sanitize or alter raw javascript rendered without using the appropriate method.
```